[2025-06-27T06:59:20.405+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T06:59:20.691+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_prod.start_task scheduled__2025-06-19T09:00:00+00:00 [queued]>
[2025-06-27T06:59:20.753+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_prod.start_task scheduled__2025-06-19T09:00:00+00:00 [queued]>
[2025-06-27T06:59:20.755+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T06:59:20.925+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): start_task> on 2025-06-19 09:00:00+00:00
[2025-06-27T06:59:21.162+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3301) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T06:59:21.185+0000] {standard_task_runner.py:72} INFO - Started process 3341 to run task
[2025-06-27T06:59:21.216+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0_prod', 'start_task', 'scheduled__2025-06-19T09:00:00+00:00', '--job-id', '940', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpup_020et']
[2025-06-27T06:59:21.265+0000] {standard_task_runner.py:105} INFO - Job 940: Subtask start_task
[2025-06-27T06:59:22.538+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0_prod.start_task scheduled__2025-06-19T09:00:00+00:00 [running]> on host e7fdb6bf901c
[2025-06-27T06:59:22.846+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0_prod' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-06-19T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-19T09:00:00+00:00'
[2025-06-27T06:59:22.870+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T06:59:22.891+0000] {logging_mixin.py:190} INFO - start
[2025-06-27T06:59:22.892+0000] {logging_mixin.py:190} INFO - ds: 2025-06-19
[2025-06-27T06:59:22.893+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-27T06:59:22.942+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T06:59:22.943+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0_prod, task_id=start_task, run_id=scheduled__2025-06-19T09:00:00+00:00, execution_date=20250619T090000, start_date=20250627T065920, end_date=20250627T065922
[2025-06-27T06:59:23.026+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T06:59:23.260+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T06:59:23.268+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
